BFME1/WinFormsMainGUI.cs

```diff
@@ -540,7 +540,7 @@ private void PiBThemeSwitcher_Click(object sender, EventArgs e)
             }
         }
 
-        private async Task InstallUpdatRepairRoutine(string ZIPFileName, string DownloadUrl, string CorrectMD5HashValue)
+        private async Task InstallUpdatRepairRoutine(string ZIPFileName, string[] DownloadUrls, string CorrectMD5HashValue)
```

Comment:
Btw this function should 100% be in the Helper class rather than slightly modified per game

Reply:
Youre correct!